onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -957,6 +1089,402 @@ Status TreeEnsembleCommonClassifier<InputType, ThresholdType, OutputType>::compu
   return Status::OK();
 }
 
+template <typename IOType, typename ThresholdType>
+class TreeEnsembleCommonV5 : public TreeEnsembleCommon<IOType, ThresholdType, IOType> {
```

Comment:
General comment for this class. It has no data, should the functions be static?
Also, the function signatures are huge, it is hard to read and probably expensive to call. Is it possible, to bind at
least some of those arguments to the instance so they are not passed as args each time?